src/ansys/fluent/core/fluent_connection.py

```diff
@@ -333,6 +335,8 @@ def __init__(
             The corresponding remote instance when Fluent is launched through
             PyPIM. This instance will be deleted when calling
             ``Session.exit()``.
+        file_transfer_service : optional
+            File transfer service. Uploads/downloads files to/from the server.
```

Comment:
```suggestion
            File transfer service for uploading files to and
            downloading files from the server.
```
Avoid using slashes and meaning is often ambiguous.

Reply:
Done. Thank you.